Il2CppInterop.Runtime/InteropTypes/Il2CppObjectBase.cs

```diff
@@ -1,26 +1,28 @@
 using System;
-using System.Reflection;
-using System.Reflection.Emit;
 using System.Runtime.CompilerServices;
 using System.Runtime.InteropServices;
-using System.Runtime.Serialization;
 using Il2CppInterop.Runtime.Runtime;
 
 namespace Il2CppInterop.Runtime.InteropTypes;
 
 public class Il2CppObjectBase
```

Comment:
Il2CppObjectBase is being removed in v2 in lieu of injecting code directly into `Il2CppSystem.Object`. As much code as possible should be implemented with extension methods or other helper classes.

Reply:
what's wrong with `Il2CppObjectBase` 😭? where will the new `Il2CppSystem.Object` be defined? will it still have behaviour i can configure (like the finalizer, and initialisation)?